specifyweb/frontend/js_src/lib/localization/preferences.ts

```diff
@@ -1413,4 +1413,7 @@ export const preferencesText = createDictionary({
     'fr-fr': 'Utiliser des infobulles modernes',
     'uk-ua': 'Використовувати сучасні підказки',
   },
+  displayOriginalAttachment: {
+    'en-us': 'Display original in preview',
```

Comment:
Can this be instead a clearer preference:

<img width="888" alt="image" src="https://user-images.githubusercontent.com/37256050/234746237-ff45479d-6f36-4e20-b65c-155a1ac7254f.png">

Something that looks and works like the one above. This communicates better what this does to the user.